tests/manage/pv_services/test_del_mon_service_and_create_pvc.py

```diff
@@ -0,0 +1,212 @@
+            svc_obj.delete(resource_name=name)
+
+            # Edit the cm
+            log.info(f"Edit the configmap {constants.ROOK_CEPH_MON_ENDPOINTS}")
+            configmap_obj = OCP(
+                kind=constants.CONFIGMAP,
+                namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            )
+            output_get = configmap_obj.get(
+                resource_name=constants.ROOK_CEPH_MON_ENDPOINTS
+            )
+            new_data = output_get["data"]
+            new_data["csi-cluster-config-json"] = (
+                new_data["csi-cluster-config-json"].replace(f'"{mon_endpoint}",', "")
+                if new_data["csi-cluster-config-json"].find(f'"{mon_endpoint}",') != 1
+                else new_data["csi-cluster-config-json"].replace(
+                    f',"{mon_endpoint}"', ""
+                )
+            )
+            new_data["data"] = ",".join(
+                [
+                    value
+                    for value in new_data["data"].split(",")
+                    if f"{mon_id}=" not in value
+                ]
+            )
+            new_data["mapping"] = (
+                new_data["mapping"].replace(f'"{mon_id}":null,', "")
+                if new_data["mapping"].find(f'"{mon_id}":null,') != -1
+                else new_data["mapping"].replace(f',"{mon_id}":null', "")
+            )
+            params = f'{{"data": {json.dumps(new_data)}}}'
+            log.info(f"Removing {mon_id} entries from configmap")
+            configmap_obj.patch(
+                resource_name=constants.ROOK_CEPH_MON_ENDPOINTS,
+                params=params,
+                format_type="strategic",
+            )
+            log.info(
+                f"Configmap {constants.ROOK_CEPH_MON_ENDPOINTS} edited successfully"
+            )
+
+            # Scale up rook-ceph-operator
+            log.info("Scale up rook-ceph-operator")
+            assert modify_deployment_replica_count(
+                deployment_name="rook-ceph-operator", replica_count=1
+            ), "Failed to scale up rook-ceph-operator to 0"
+            log.info("Successfully scaled up rook-ceph-operator to 1")
+            log.info("Validate rook-ceph-operator pod is running")
+            POD_OBJ.wait_for_resource(
+                condition=constants.STATUS_RUNNING,
+                selector=constants.OPERATOR_LABEL,
+                resource_count=1,
+                timeout=600,
+                sleep=5,
+            )
+
+            # Validate all mons are running
+            log.info("Validate all mons are up and running")
+            POD_OBJ.wait_for_resource(
+                condition=constants.STATUS_RUNNING,
+                selector=constants.MON_APP_LABEL,
+                resource_count=mon_count,
+                timeout=1200,
+                sleep=5,
+            )
+            log.info("All mons are up and running")
+
+            # Check the ceph health OK
+            ceph_health_check(tries=90, delay=15)
```

Comment:
Please also check for storage pods status, we need to make sure all the storage pods are healthy after mon recovery.

Reply:
done, added now